lua/entities/gmod_wire_pod.lua

```diff
@@ -108,6 +108,13 @@ function ENT:Initialize()
 	self:SetUseType( SIMPLE_USE )
 
 	table.insert(pods, self)
+	if #pods == 1 then
+		hook.Add("PlayerBindDown", "gmod_wire_pod", PlayerBindDownHook)
```

Comment:
You'll need these functions' locals pre-defined at the top of the file.

```lua
local PlayerBindDownHook

---------

function PlayerBindDownHook()
end
```

Reply:
Good catch! Should be fixed now, sorry about that.